src/dataconnect/graphqlError.ts

```diff
@@ -1,5 +1,5 @@
 import { GraphqlError } from "./types";
-const Table = require("cli-table");
+const Table = require("cli-table3");
```

Comment:
This most likely should be cli-table, not cli-table3 - cli-table3 looks like a transitive dependency (since it is only listed in shrinkwrap, not in package.json) and therefore we can't use it directly. 

Node is weird, and this will work on you local mahcine, but will break once you ship it. We have a linter check to catch these, but apparently it only catches imports, not requires

Reply:
Ahh I changed it to `cli-table3` in order to use the `wordWrap` functionality (which doesn't look like it's available in `cli-table`), otherwise the error message and workarounds in the table are too long for the table to render properly.

Could I add `cli-table3` into package.json? Or is there a better way within `cli-table` to format things properly? FWIW from https://github.com/cli-table/cli-table3/tree/master it looks like `cli-table` is unmaintained and `cli-table3` is the current version.

Reply:
Oooh, if they are API compatible lets just get rid of our dependency on cli-table and use cli-table3. Would you be willing to upgrade it across the codebase so that we don't have 2 redundant deps?

Reply:
Done in #8176